repondeur/zam_repondeur/templates/login.html

```diff
@@ -8,8 +8,11 @@ <h1>Entrer dans Zam</h1>
         {% if request.session.pop('missing_email', None) %}
             <p class="error">La saisie d’une adresse de courriel est requise.</p>
         {% endif %}
+        {% if request.session.pop('already_in_use', None) %}
+            <p class="error">Cette addresse de courriel est déjà utilisée<br>dans une autre équipe.</p>
+        {% endif %}
         <label for="email">Votre courriel</label>
-        <input type="email" name="email" autocomplete="email" required autocorrect="off" spellcheck="false">
```

Comment:
Why not anymore required?

Reply:
Just a test, should not have been committed. Thanks for catching it!